### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by bastianeicher** [spectreconsole/spectre.console#368]
Hi there,

I've been using Spectre.Console v0.38.0 in my project [Zero Install](https://github.com/0install/0install-dotnet) and it's working great! 👍
Now I'd love to use some of the new features on `main` such as indeterminate progress (6f16081f427276f04ba2ae9d971db78aede44f31).

The recently added NuGet packages on GitHub Packages were very helpful for testing the latest code beforehand. However, I'd prefer not to depend on pre-release packages in external projects, so I was wondering if you could publish a new release on NuGet.org.

Regards
Bastian

**Comment by Frassle** [comment]
It would also be good to have a documented process for when and how new release will be made, so that issues like this don't need to be raised in the future.

**Comment by phil-scott-78** [comment]
We just pushed a change to the guts of the library with PR #372. Assuming no bugs pop up in the next day it's a candidate for a full release. 

Regarding future releases, @patriksvensson and I have discussed some form of tracking for this. Since it's just the two of us working in our spare time, I don't think we can ever promise a consistent cadence any time soon. But we'd like to at least get a list of bugs and features that we need to get through before we finalize a release.
